### PR TITLE
Fixes saving of default assigned policy

### PR DESCRIPTION
**Pull request by dev-gh**
On assigning of default policy to application there were no saving
of RequestTypes and AppHMIType parameters.

Relates-to: APPLINK-25284

### CODE REVIEW

**Comment by dev-gh** [comment]
@VProdanov @AByzhynar @OHerasym @AGaliuzov 
Please review

**Comment by OHerasym** [comment]
Reviewed

File: src/components/policy/src/sql_pt_representation.cc [hunk 1]
```diff
@@ -1563,11 +1563,21 @@ bool SQLPTRepresentation::SetDefaultPolicy(const std::string& app_id) {
...
+  if (!GatherRequestType(kDefaultId,&request_types) ||
+      !SaveRequestType(app_id, request_types)) {
+    return false;
+  }
+  policy_table::AppHMITypes app_types;
+  if (!GatherAppType(kDefaultId, &app_types) ||
+      !SaveAppType(app_id, app_types))  {
+    return false;
+  }
+  return SetIsDefault(app_id, true);;
```
Comment (AByzhynar) [inline src/components/policy/src/sql_pt_representation.cc]: @dev-gh Two semi-colons art the line end

**Comment by AByzhynar** [comment]
@dev-gh Reviewed. Please check coding style

**Comment by dev-gh** [comment]
@AByzhynar 
fixed with https://github.com/smartdevicelink/sdl_core/pull/681/commits/383cbfdfdc8f42051f05575cdc93654548b97297
Also unit test has updated. Please check.

**Comment by VProdanov** [comment]
Reviewed

**Comment by AGaliuzov** [comment]
Reviwed

**Pull request merged**